plugins/benchmark.yml

```diff
@@ -0,0 +1,6 @@
+pluginName: benchmark
+version: v0.1.2
+repository: https://github.com/tamirdavid/benchmark
+maintainers:
+  - tamirdavid
+tag: v0.1.2
```

Comment:
It looks like ythe plugin's repository doesn't have a tag named `v0.1.2`. You'll need to create this tag for the registry to be able to build your pugin.


![image](https://user-images.githubusercontent.com/7830056/218728136-253748f1-8303-45f9-86f1-d3e2be688898.png)

Reply:
@eyalbe4 added